src/patternfly/components/ChipGroup/chip-group-action.hbs

```diff
@@ -0,0 +1,6 @@
+<button class="pf-c-chip-group__action{{#if chip-group-action--modifier}} {{chip-group-action--modifier}}{{/if}}"
```

Comment:
@mcoker what was your reasoning behind wanting to create a new element in the chip group vs using pf-c-button?

Reply:
The previous selector used `.pf-c-chip-group + .pf-c-button` (or something like that) to target the close all button. Ideally we wouldn't use the sibling selector since it can be fragile, and would not style the button (a component in a component), and instead just use a BEM class selector that's part of the chip-group component to style it. Also that doesn't scale if we end up adding another `.pf-c-button` later in that same spot that does something else.